editor/plugins/tiles/tile_atlas_view.cpp

```diff
@@ -311,6 +311,7 @@ RID TileAtlasView::_get_canvas_item_to_draw(const TileData *p_for_data, const Ca
 		RID ci_rid = RS::get_singleton()->canvas_item_create();
 		RS::get_singleton()->canvas_item_set_parent(ci_rid, p_base_item->get_canvas_item());
 		RS::get_singleton()->canvas_item_set_material(ci_rid, mat->get_rid());
+		RS::get_singleton()->canvas_item_set_default_texture_filter(ci_rid, RS::CANVAS_ITEM_TEXTURE_FILTER_NEAREST);
```

Comment:
This should make sure things stay consistent, even if we choose to change the filter (maybe dynamically) later on:
```suggestion
		RS::get_singleton()->canvas_item_set_default_texture_filter(ci_rid, RS::CanvasItemTextureFilter(p_base_item->get_texture_filter_in_tree()));
```